src/components/Avatar/Avatar.jsx

```diff
@@ -49,6 +54,8 @@ const Avatar = ({
     return customSize ? { height: `${customSize}px`, width: `${customSize}px` } : {};
   }, [customSize]);
 
+  ariaLabel = ariaLabel || (typeof tooltipProps?.content === "string" ? tooltipProps.content : undefined);
```

Comment:
If someone pass tooltipProps.content is should "win" the aria label, because the aria label is the default (only if there is no other configuration)

Reply:
we talk yesterday about ariaLabel not need to take their value from the tooltip content in any case